guide/english/css/fonts/index.md

```diff
@@ -101,12 +100,24 @@ p.small {
 Font properties can be specified with the shorthand font. 
 It takes as value (in this order):
 - font-style (optional)
+`p {font-style: italic;}`
```

Comment:
Could you please move these points to a new line. the style name and declaration look too crowded on one line